api/schemas/HealthInfo.yml

```diff
@@ -0,0 +1,23 @@
+type: object
+properties:
+  status:
+    type: string
+    description: Service status.
+    enum:
+      - pass
+  version:
+    type: string
+    description: Service version.
+    example: 0.0.1
+  commit:
```

Comment:
We cannot leave `commit`. We must adhere to standard. If you have fields that are missing, we probably must work with standardization body and argument why we need those, so that they can rectify in the next version pf the draft. Otherwise please find the closest field and put commit information there (probably `releaseId`).

Reply:
ok, where I put `build_time`?

Reply:
That's a good question. Let me check.

Reply:
@manuio I have sent an e-mail to the author. Maybe we'll have to open an issue [here](https://github.com/inadarei/rfc-healthcheck/issues) as well. Let's give it a day or two, and if we have no answers, I will merge the current PR.

Reply:
I have opened an issue as well: https://github.com/inadarei/rfc-healthcheck/issues/76